src/pages/bubblegum-v2/collections.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Verifying Collections
+metaTitle: Verifying Collections | Bubblegum V2
+description: Learn how to set, verify and unverify collections on Bubblegum
+---
+
+A cNFT can be added to a MPL-Core Collection either on mint or later. {% .lead %}
+
+If you are not familiar with the concept of collections with regard to NFTs, they are special non-compressed NFTs that can be used to group other NFTs together. The data of the **Collection** is therefore used to describe the name and the branding of the entire collection. Since Bubblegum V2 it also allows additional features on collection level, like allowing freezing and thawing of cNFTs by a delegate without the need of interaction by the leaf owner. You can [read more about MPL-Core Collections here](/core/collections).
+
+Note that it is possible to mint a Compressed NFT directly into a collection by using the **MintV2** instruction [documented here](/bubblegum-v2/mint-cnfts#minting-to-a-collection). That being said, if you have already minted a cNFT without a collection, let's see how we can set the collection on that cNFT. Other to bubblegum v1, which uses Metaplex Token Metadata Collections which have a "verified" boolean, Bubblegum V2 uses MPL-Core Collections which do not have that boolean.
```

Comment:
wdyt about saying something more like its the asset which has the concept of verified collection or not, not the collection itself.  And its that Bubblegum V2 assets do not have the concept of a verified flag.

Although the funny thing is that in DAS it has a verified flag still, but that's besides the point.

Reply:
I feel like since we are talking bubblegum adding more information about "verified" stuff just makes it unnecessary complicated - we should just ignore it. Iirc no RPC shows `verified: false` if you dont request it.